src/utils.js

```diff
@@ -15,27 +15,32 @@ export function toPosixPath(modulePath) {
   return modulePath.replace(/\\/g, '/');
 }
 
+export function isRelativePath(nodePath) {
+  return nodePath.match(/\.?\.\//);
```

Comment:
Should we start the regex with `^` to make sure we match only the beginning?

Reply:
Ah, good catch.